runtime/moonbase/src/lib.rs

```diff
@@ -202,6 +203,21 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	state_version: 0,
 };
 
+/// We need to duplicate this because the `runtime_version` macro is conflicting with the
+/// conditional compilation at the state_version field.
+#[cfg(not(feature = "runtime-benchmarks"))]
+#[sp_version::runtime_version]
+pub const VERSION: RuntimeVersion = RuntimeVersion {
+	spec_name: create_runtime_str!("moonbase"),
+	impl_name: create_runtime_str!("moonbase"),
+	authoring_version: 4,
+	spec_version: 3200,
+	impl_version: 0,
+	apis: RUNTIME_API_VERSIONS,
+	transaction_version: 2,
+	state_version: 1,
+};
+
```

Comment:
Instead of duplicating the whole `VERSION` struct, lets create a `STATE_VERSION` constant and then use it inside `VERSION`.

```rs
#[cfg(feature = "runtime-benchmarks")]
const STATE_VERSION: u8 = 0;
#[cfg(not(feature = "runtime-benchmarks"))]
const STATE_VERSION: u8 = 1;

#[sp_version::runtime_version]
pub const VERSION: RuntimeVersion = RuntimeVersion {
	spec_name: create_runtime_str!("moonbase"),
	impl_name: create_runtime_str!("moonbase"),
	authoring_version: 4,
	spec_version: 3200,
	impl_version: 0,
	apis: RUNTIME_API_VERSIONS,
	transaction_version: 2,
	state_version: STATE_VERSION
}
```

Reply:
It's not possible, I guess something related to `#[sp_version::runtime_version]` macro requiring state_version to be explicit number.